include/crypto/evp.h

```diff
@@ -52,6 +52,21 @@ struct evp_pkey_ctx_st {
         } ciph;
     } op;
 
+    /*
+     * Cached parameters.  Operation inits should call
+     * evp_pkey_ctx_unleash_delayed_data() when the operation has been set
```

Comment:
dragons?

Reply:
...  I really should, just for your pleasure :wink:

Reply:
Fixed

Reply:
Warning: here be dragons.